pkg/network/node/networkpolicy.go

```diff
@@ -822,7 +823,8 @@ func (np *networkPolicyPlugin) parsePeerFlows(npns *npNamespace, npp *npPolicy,
 					sel, _ := metav1.LabelSelectorAsSelector(peer.NamespaceSelector)
 					if _, ok := np.nsMatchCache[sel.String()].matches[HostNetworkNamespace]; ok {
 						for _, network := range np.node.networkInfo.ClusterNetworks {
-							cidrIP := network.ClusterCIDR.IP
+							cidrIP := make(net.IP, len(network.ClusterCIDR.IP))
+							copy(cidrIP, network.ClusterCIDR.IP)
```

Comment:
hm... I feel like there ought to be an easier way to do this, but I can't think of one really...